Description: fix FTBFS on GNU Hurd by explicitly linking librt and libdl
Author: Jonathan Wiltshire <jmw@debian.org>
Origin: Debian
Forwarded: https://github.com/inspircd/inspircd/issues/98
Last-Update: 2012-04-17

--- inspircd-2.0.5.orig/make/template/main.mk
+++ inspircd-2.0.5/make/template/main.mk
@@ -41,6 +41,9 @@ INSTMODE_LIB = 0644
 @IFEQ $(SYSTEM) gnukfreebsd
   LDLIBS += -ldl -lrt
 @ENDIF
+@IFEQ $(SYSTEM) gnu
+  LDLIBS += -ldl -lrt
+@ENDIF
 @IFEQ $(SYSTEM) solaris
   LDLIBS += -lsocket -lnsl -lrt -lresolv
 @ENDIF
